realm/realm-library/src/androidTest/kotlin/io/realm/KotlinSchemaTests.kt

```diff
@@ -49,7 +49,7 @@ class KotlinSchemaTests {
 
     @Test
     fun kotlinTypeNonNull() {
-        val objSchema = realm.schema.get(AllKotlinTypes::class.simpleName)!!
+        val objSchema = realm.schema.get(AllKotlinTypes::class.simpleName!!)!!
 
         // Document current nullability. Ideally all should be non-nullable. This is currently
```

Comment:
I guess that this comment means we should respect nullability information in Kotlin's type information.

Currently users need to add `@Required` annotation to define non-null field even if the type of the field is non-null (e.g. `String`, not `String?`).
Once we implement that in our annotation processor, fields below are all non-null.